commands/instances.go

```diff
@@ -237,111 +240,125 @@ func Init(req *rpc.InitRequest, responseCallback func(r *rpc.InitResponse)) erro
 		})
 	}
 
-	loadBuiltinTools := func() []error {
-		builtinPackage := pmb.Packages.GetOrCreatePackage("builtin")
-		return pmb.LoadToolsFromPackageDir(builtinPackage, pmb.PackagesDir.Join("builtin", "tools"))
-	}
+	{
```

Comment:
maybe would be better to use a real function instead of `{` and indenting all the content of it

Reply:
I tried, we need to pass a lot of local variables as parameters, it's not worth it IMHO.